db/migrate/20251024015907_populate_image_quality_setting.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+class PopulateImageQualitySetting < ActiveRecord::Migration[8.0]
+  def up
```

Comment:
I think we should add a migration test here so that we can easily verify that this migration is working as intended.